### PR TITLE
Support Zeebe 1.1

### PR DESCRIPTION
**Pull request by barmac**
Adds support for:

* bpmn:SendTask
* bpmn:ScriptTask
* bpmn:BusinessRuleTask

Related to camunda/camunda-modeler#2296
Related to camunda/camunda-modeler#2297
Related to camunda/camunda-modeler#2298

### CODE REVIEW

File: test/camunda-cloud/features/popup-menu/ReplaceMenuProviderSpec.js [hunk 1]
```diff
@@ -483,6 +543,19 @@ describe('camunda-cloud/features - PopupMenu', function() {
 
 // helper //////////
 
+/**
+ * Query popup menu to check if queries exist.
+ *
+ * @param {string[]} entries - ids of the expected entries
+ */
+function expectEntries(popupMenu, entries) {
```
Comment (nikku) [inline test/camunda-cloud/features/popup-menu/ReplaceMenuProviderSpec.js:551]: :+1: for this.

**Review by nikku — APPROVED** [review]
Works for me.

**Comment by nikku** [comment]
I find it odd that coverage is not reported here.

**Comment by nikku** [comment]
I believe we can release this one, right? Or is there more stuff we're waiting for?

**Comment by barmac** [comment]
Not from my side. I will release it today.

**Comment by nikku** [comment]
Actually there is one thing I'm following up with: https://github.com/camunda/camunda-modeler/issues/2267.

**Pull request merged**